### ISSUE TITLE
Cannot save provisioning settings

### ISSUE THREAD

**Issue by radek3911** [nextcloud/mail#7426]
### Steps to reproduce

1. I installed Docker AIO on ARM
2. I installed mail plugin
3. I tried to create new provisioning settings, but when I tried to save nothing happens. There is no network request in network tab.

### Expected behavior

save provisioning settings

### Actual behavior

nothing happens - I click and the app does not react, no error message

### Mail app version

1.14.1

### Mailserver or service

Generic IMAP (email.cz)

### Operating system

Host: Ubuntu, but NextCloud is running in AIO docker

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database

PostgreSQL

### Additional info

Host server has ARM Ampere processor, but I think it's frontend clientside issue.

**Comment by radek3911** [comment]
https://user-images.githubusercontent.com/15276167/195722580-051dba6a-3e75-4572-8d90-2cdd98ea62c9.mp4

**Comment by ChristophWurst** [comment]
Do you get an error(s) in the browser console?

**Comment by radek3911** [comment]
Hello, no errors in console. I have noticed return event in debugger is null. I will attach more details later today.

Also just so I understand this feature correctly:

I want to "prefill" email config when user first logins to new account (do that they only have to fill their email and password) is this correct use of the feature?

Can I use autodiscovery as a workaround?

Regards,
Radek

**Comment by radek3911** [comment]
Also tried on Firefox, Brave and chrome and had same result everywhere...

**Comment by ChristophWurst** [comment]
Yes, that is how the feature works. You as an admin can configure the IMAP accounts for your users.

**Comment by radek3911** [comment]
Hi, PFA video.

https://user-images.githubusercontent.com/15276167/195850061-6d3efe7f-c2e1-4dc0-b3bf-4a3abb843301.mp4

**Comment by ChristophWurst** [comment]
Thanks. That makes it more clear.

**Comment by prismopensource** [comment]
Same problem over here with PHP 8.1 and latest stable Nextcloud and Mail app.

**Comment by nokia001** [comment]
Same here.
NC 23.0.10 and 24.0.7
Mail  1.14.2
PHP:  8.0.14

**Comment by PaperTowels207** [comment]
I had this problem when I moved to 1.14.1 initially. However, my config from 1.13.9 was still there after the update, I just cannot modify it using the gui, and adding a new one would not save. The debugging, browser console, and httpd logs did not show anything. It felt like the button did not have a corresponding backend, but that's just a wild guess. I was able to make changes to my existing config by modifying the OC table directly in the interim.
